config/example-generic-caretesian.cfg

```diff
@@ -0,0 +1,138 @@
+# This file is an example config file for cartesian style printers.
+# One may copy and edit this file to configure a new cartesian
+# printer.
+
+# DO NOT COPY THIS FILE WITHOUT CAREFULLY READING AND UPDATING IT
+# FIRST. Incorrectly configured parameters may cause damage.
+
+# See docs/Config_Reference.md for a description of parameters.
+
+[carriage x]
+position_endstop: 0
+position_max: 300
+homing_speed: 50
+endstop_pin: ^PE5
+
+[carriage y]
+position_endstop: 0
+position_max: 200
+homing_speed: 50
+endstop_pin: ^PJ1
+
+[extra_carriage y1]
+primary_carriage: y
+endstop_pin: ^PB6
+
+[carriage z]
+position_endstop: 0.5
+position_max: 100
+endstop_pin: ^PD3
+
+[dual_carriage u]
+primary_carriage: x
+position_endstop: 300
+position_max: 300
+homing_speed: 50
+endstop_pin: ^PE4
+
+[stepper x]
+kinematics: x+y
+step_pin: PF0
+dir_pin: PF1
+enable_pin: !PD7
+microsteps: 16
+rotation_distance: 40
+
+[stepper u]
```

Comment:
Not a "blocker", but if a config name can be any arbitrary name then I think it would be nice for the example configs to make that more clear - for example:
```
[extra_carriage my_extra_y_carriage]
...
[dual_carriage my_idex_carriage]
...
[stepper my_stepper_for_x]
...
[stepper my_idex_stepper]
...
[stepper my_secondary_idex_stepper]
```

In some cases carriage names must be a pre-defined value (eg, "[carriage z]"), but in other cases the user can specify a name of their choice, and making that a little more clear in the examples may help.

Reply:
I updated the stepper names to highlight the possibility. For the carriage names, I'm a bit unsure, in a sense that in many cases these documentation snippets are copied by people to adjust them to their needs, but keep the main structure as-is. So I do not want to promote too long carriage names, since they are used later in the stepper configuration, and it looks a bit weird when long names appear there.